fancycompleter.py

```diff
@@ -345,13 +345,10 @@ def has_leopard_libedit(config):
     # Adapted from IPython's rlineimpl.py.
     if config.using_pyrepl or sys.platform != 'darwin':
         return False
-    from subprocess import Popen, PIPE
-    cmd = ["otool", "-L", config.readline.__file__]
-    p = Popen(cmd, stdout=PIPE, stderr=PIPE)
-    stdout, stderr = p.communicate()
-    if p.returncode == 0 and 'libedit' in stdout.decode('utf-8'):
-        return True
-    return False
+
+    # Official Python docs state that 'libedit' is in the docstring for
+    # libedit readline
```

Comment:
```suggestion
    # libedit readline.
```

Reply:
Would have committed this myself, but do not have perms here - see the "allow maintainer edits" checkbox on the right.

Reply:
![Selection_191](https://user-images.githubusercontent.com/33840/58432823-56179280-8082-11e9-9e03-2c6b28749eaf.png)

:O it's checked on my screen. regardless, I'll push it in